Tools/setup/ubuntu.sh

```diff
@@ -159,10 +159,12 @@ if [[ $INSTALL_NUTTX == "true" ]]; then
 	fi
 
 	# arm-none-eabi-gcc
-	NUTTX_GCC_VERSION="9-2020-q2-update"
-	NUTTX_GCC_VERSION_SHORT="9-2020q2"
+	NUTTX_GCC_VERSION="10-2020-q4-major"
```

Comment:
Note: I was using ARM GCC 10 for a while and did not have any issue until this one time where I had to flash a v5x board for a customer demo and it didn't boot anymore. After switching back to GCC 9 it was working again.

Maybe that's all in the past but I still wanted to mention this experience.

Reply:
Haven't seen that but that's certainly odd.

Reply:
Hey @MaEtUgR any guidance so I can repro this? hopefully it's gone

Reply:
I would actually go with 10.3-2021.10 from:

https://developer.arm.com/-/media/Files/downloads/gnu-rm/10.3-2021.10/gcc-arm-none-eabi-10.3-2021.10-x86_64-linux.tar.bz2

Also see: https://developer.arm.com/downloads/-/gnu-rm